### PR TITLE
feat(groupChats): add group events AP-1174

### PR DESCRIPTION
**Pull request by pavlzk**
**What this PR does** 📖
Add event subscriptions to handle group updates

**Which issue(s) this PR fixes** 🔨
AP-1174

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: store/groups/mutations.test.ts [hunk 1]
```diff
@@ -100,52 +102,74 @@ describe('mutations.default.addGroup', () => {
...
+  //
+  // test('4', () => {
+  //   const result: any = mutations.default.setGroupMembers(state(), '9876')
+  //   expect(result).toMatchSnapshot()
+  // })
+  //
+  // test('5', () => {
+  //   const result: any = mutations.default.setGroupMembers(state(), '')
+  //   expect(result).toMatchSnapshot()
+  // })
```
Comment (stavares843) [inline store/groups/mutations.test.ts:174]: why is this commented out? 🔨

Comment (pavlzk) [reply store/groups/mutations.test.ts:174]: I updated several tests that I think are enough and commented those two in case QA still want to have more

Comment (stavares843) [reply store/groups/mutations.test.ts:174]: you can remove these then 🔨 thanks for updating the tests 🔨

Comment (pavlzk) [reply store/groups/mutations.test.ts]: removed

**Review by Jekrimo — APPROVED** [review]

**Review by RetroPronghorn — APPROVED** [review]

**Comment by stavares843** [comment]
missing fixing conflicts 🔨

**Comment by stavares843** [comment]
@pavlzk 

image


this is now 222 files changed, because its including commits from other PR's 🔨

**Comment by pavlzk** [comment]
> @pavlzk
> 
> image
> 
> this is now 222 files changed, because its including commits from other PR's 🔨

Yes. I'm trying to resolve it

**Comment by phillsatellite** [comment]
Tested after rebase: Seems to be a new crash on just this branch and on Chrome, Safari, and Brave (I did verify that this does not happen on dev branch) 
Flow for crash:
1. Create new account 
2. Add friend 
3. Send message in chat
4. Crash - Cannot read properties of undefined (reading ")

https://user-images.githubusercontent.com/93608357/162038892-e5119830-1562-4793-a799-8d352d5bb2ee.mov

**Comment by pavlzk** [comment]
> Tested after rebase: Seems to be a new crash on just this branch and on Chrome, Safari, and Brave (I did verify that this does not happen on dev branch) Flow for crash:
> 
> 1. Create new account
> 2. Add friend
> 3. Send message in chat
> 4. Crash - Cannot read properties of undefined (reading ")
> 
>  crash.after.rebase.mov

Thanks! Fix pushed. Could you check ?

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**